packages/xrpl/test/models/utils.test.ts

```diff
@@ -207,5 +208,43 @@ describe('Models Utils', function () {
       assert.isUndefined(parsed.lsfDisallowIncomingTrustline)
       assert.isUndefined(parsed.lsfAllowTrustLineClawback)
     })
+
+    it('parseTransactionFlags all enabled', function () {
+      const tx: PaymentChannelClaim = {
+        Account: 'r...',
+        TransactionType: 'PaymentChannelClaim',
+        Channel:
+          'C1AE6DDDEEC05CF2978C0BAD6FE302948E9533691DC749DCDD3B9E5992CA6198',
+        Flags: {
```

Comment:
We should add a test case for when Flags is a number. (e.g. [this transaction](https://livenet.xrpl.org/transactions/C1356ED3033210B0D153D7CDFBD4BF1076858E430E8ABA6D34C942C17051A5E0/raw)

Reply:
added